File: src/Readerz/ClientApp/src/app/card/card.ts
export class Card {
    constructor(
        public id?: number, 
        public front?: string, 
        public back?: string) {

    }

    copy(card: Card) {
        this.id = card.id;
        this.front = card.front;
        this.back = card.back;
    }
}

File: src/Readerz.Domain/Entities/Card.cs
using Readerz.Web.Domain.Common;

namespace Readerz.Domain.Entities
{
    public class Card : AuditableEntity
    {
        public int Id { get; set; }
        public string Front { get; set; }
        public string Back { get; set; }

        public int CardSetId { get; set; }
        public CardSet CardSet { get; set; }
    }
}
